CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 *It is strongly advised to perform an update of your tentacles after updating OctoBot. (start.py tentacles --install --all)*
 
+## [1.0.11] - 2024-07-05
```

Comment:
Should we make a special version for these new features like 1.1.0 or 2.0.0?

Reply:
Good idea, let's go for 1.1.0 !

Reply:
up